index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="container">
+        <header>
+            <div class="right-side">
+                <h3>Anicet Fantômas</h3>
+            </div>
+            <div class="left-side">
+                <div class="menu-lines">
+                    <div class="menu-line"></div>
+                    <div class="menu-line"></div>
+                    <div class="menu-line"></div>
+                </div>
+            </div>
+        </header>
+        <section>
+            <h1>Hey There.<br>I'm Anicet</h1>
+            <h2>I'am a Software Developer</h2>
+            <p>I can help you build a product. featuer or <br> website Look through some of my work <br> and experience!
+                if you see <br>
+                and have a project you need coded. <br> don't hestiate to contact me.
+            </p>
+            <div class="footer-images">
+                <img src="images/Vector3.png" alt="" id="footer-img">
+                <img src="images/Vector2.png" alt="" id="footer-img">
+                <img src="images/Vector1.png" alt="" id="footer-img">
+                <img src="images/Vector.png" alt="" id="footer-img">
+            </div>
```

Comment:
- [ ] Please update this list, the list of icons should use a `ul` tag to group them, and `li` per icon; each icon should have a link (a tag). Eg: 
```
 <ul>
    <li><a href="#"><img src="images/Vector3.png" alt="" id="footer-img"></a></li>
 </ul>
```

- [ ] Please add all icons as listed in Figma, and evenly align them 😉:

Figma  | Yours
------------- | -------------
<img width="377" alt="Screenshot 2022-02-10 at 22 25 20" src="https://user-images.githubusercontent.com/77038610/153472398-58fd894b-4e1f-44ce-b5d4-7c0ec8e5df57.png"> | <img width="423" alt="Screenshot 2022-02-10 at 22 24 56" src="https://user-images.githubusercontent.com/77038610/153472346-2009d768-9477-440d-bd2e-29de1923c6f6.png">